src/main/java/com/larvalabs/svgandroid/ParserHelper.java

```diff
@@ -18,24 +17,14 @@
  */
 public class ParserHelper {
 
-	private static final Field STRING_CHARS;
-	static {
-		try {
-			STRING_CHARS = String.class.getDeclaredField("value");
-			STRING_CHARS.setAccessible(true);
-		} catch (Exception e) {
-			throw new RuntimeException(e);
-		}
-	}
-
 	private final char[] s;
 	private final int n;
 	private char current;
 	public int pos;
 
 	public ParserHelper(String str, int pos) {
 		try {
-			this.s = (char[]) STRING_CHARS.get(str);
+			this.s = str.toCharArray();
```

Comment:
This may be costly, because toCharArray() copies the characters,as the javadoc says. This object might be constructed many even during processing a single image. I have changed s to be a String, and use charAt() instead of getting from an array. Should charAt() be fast, it is a native method in newer SDKs, older use array access inside.
Should actually measure performance, it works as fast as previously (as perceived) for me anyway.